src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -17,6 +17,15 @@ import { ItemEditAdvancedParametersComponent } from '../item-edit-advanced-param
 import { Mode, ModeService } from 'src/app/shared/services/mode.service';
 import { readyData } from 'src/app/shared/operators/state';
 
+const INIT_DATE = '1000-01-01T00:00:00Z';
+const DEFAULT_DATE = '9999-12-31T23:59:59Z';
+const DEFAULT_DATES = [ INIT_DATE, DEFAULT_DATE ];
+
+interface InitialItemState extends Item {
```

Comment:
Why initial, it is not only valid initially. 
`FormItem`?

Reply:
Before it was for "initialFormData: Item", I took "initial" word from there.

Reply:
But that's a variable name, which defines its role. Here it is type name, which describe what this describes.

By the way, what would be cleaner IMO is to define initialFormData? of type `Item & { duration.... }` directly and transform `resetFormWith(item: InitialItemState)` to `resetForm` and use the instance variable directly.

Reply:
Fixed